Python/bytecodes.c

```diff
@@ -2092,6 +2106,11 @@ dummy_func(
             Py_DECREF(owner);
         }
 
+        macro(STORE_ATTR_SLOT) =
+            unused/1 +
+            _GUARD_TYPE_VERSION_STORE +
```

Comment:
It seems a shame that we need ` _GUARD_TYPE` and ` _GUARD_TYPE_VERSION_STORE`.
We need both because we need the name of the original instruction in `DEOPT_IF`.

We could remove the instruction name from `DEOPT_IF` and have the code generator insert it.
This would give us a bit flexibility in how we handle these exits as well.
For another PR, though.

Reply:
Yeah, this is a remnant from before the code generator. I've filed https://github.com/python/cpython/issues/109979.